### PR TITLE
feat: Add Structured Logging error capture integration test

### PR DESCRIPTION
**Pull request by Mmarzex**

### CODE REVIEW

File: .github/workflows/node-integrate.yml
Comment (Danwakeem) [inline .github/workflows/node-integrate.yml:1]: Don’t forget to reset this 😎

**Review by Danwakeem — CHANGES_REQUESTED** [review]
Looks good just don’t forget to rest the action file 👍

**Comment by Mmarzex** [comment]
hmm our integration tests seem to be running out of memory or something. Any ideas on that @Danwakeem ?

**Comment by Danwakeem** [comment]
@Mmarzex unfortunately I think you just have to wait a minute or two before the memory issue fixes itself 😢

**Comment by Mmarzex** [comment]
I did wait a while between trying again 😢

**Comment by Mmarzex** [comment]
@Danwakeem I set the integration tests to increase the javascript heap size. I think the addition of the new dependencies for the structured log tests was too much with the default.

**Review by Danwakeem — APPROVED** [review]
Looking good 😎

**Pull request merged**